src/sentry/static/sentry/app/views/settings/components/settingsBreadcrumb/types.tsx

```diff
@@ -0,0 +1,7 @@
+import {RouteProps} from 'react-router';
+
+// TODO(ts): The `name` attribute doesn't appear on any of the react router route types
```

Comment:
It comes from our `routes.jsx`, no? Is `name` an actual prop there or will we need to fix those when we convert that file to ts?

Reply:
It's not a real prop lol